SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -34,14 +35,14 @@ Volume {
 
 			server.sync;
 
-			this.updateSynth;
+			updateFunc = {
+				ampSynth = nil;
+				if(persist) { this.updateSynth }
+			};
 
-			if(updateFunc.isNil) {
```

Comment:
I can't test it here, but could you make sure that `ServerTree` doesn't accumulate these functions?

Reply:
I don't see any accumulation happening.